src/app.js

```diff
@@ -49,22 +52,19 @@ p5.Compressor = Compressor;
 import './soundRecorder';
 import './peakDetect';
 
-
-
 import Distortion from './distortion';
 p5.Distortion = Distortion;
 
-
 import Gain from './gain';
 p5.Gain = Gain;
 
-
-
-
 import AudioVoice from './audioVoice';
 p5.AudioVoice = AudioVoice;
 
 import MonoSynth from './monosynth';
 p5.MonoSynth = MonoSynth;
 
 import './polysynth';
+
+import OnsetDetect from './onsetDetect';
+p5.OnsetDetect = OnsetDetect;
```

Comment:
looks like a change from a different PR was added to this one, too! Hopefully this won't cause merge conflicts.